<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QTextEdit" name="display_screen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>721</width>
     <height>291</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="message_textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>721</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>StartServer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>CloseServer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_Label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>144</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器状态：关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
   <property name="text">
    <string>initDB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
